packages/block-library/src/post-author/edit.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * WordPress dependencies
+ */
+import { useEntityProp, useEntityId } from '@wordpress/core-data';
+
+function PostAuthorDisplay() {
+	const [ author ] = useEntityProp( 'postType', 'post', 'author' );
+	return <address>{ author }</address>;
```

Comment:
Potentially we could show a Post Author Select if the block is selected. We could start with a readonly block as well.

Reply:
We still don't have an async Combobox component for this and I don't want that to block this structural PR.

Reply:
We should use the same thing we already have for the Post Author in the document sidebar (imperfect) but I don't see this as mandatory for that PR anyway.

Reply:
Agreed

Reply:
>We should use the same thing we already have for the Post Author

Not on that though. Implementing that is so complex that it will be faster to implement the generic Combobox first and use that instead.

Reply:
In my testing, this shows the id of the author, I believe we should show the username at least and it should be shown exactly like the fontend `By %`

Reply:
Oops, the post object references author IDs. I've updated the code to fetch the actual author object and use its name instead.